P5/Source/Specs/annotation.xml

```diff
@@ -1,16 +1,16 @@
-<?xml version="1.0" encoding="UTF-8"?>
+<?xml version="1.0" encoding="UTF-8"?>
 <!--
 Copyright TEI Consortium. 
 Dual-licensed under CC-by and BSD2 licences 
 See the file COPYING.txt for details
 $Date$
 $Id$
--->
-<?xml-model href="http://jenkins.tei-c.org/job/TEIP5-dev/lastSuccessfulBuild/artifact/P5/release/xml/tei/odd/p5.nvdl" type="application/xml" schematypens="http://purl.oclc.org/dsdl/nvdl/ns/structure/1.0"?>
-<elementSpec xmlns="http://www.tei-c.org/ns/1.0" ident="annotation" mode="add" module="linking"
-  xmlns:sch="http://purl.oclc.org/dsdl/schematron">
+-->
+<?xml-model href="http://jenkins.tei-c.org/job/TEIP5-dev/lastSuccessfulBuild/artifact/P5/release/xml/tei/odd/p5.nvdl" type="application/xml" schematypens="http://purl.oclc.org/dsdl/nvdl/ns/structure/1.0"?>
+<elementSpec xmlns="http://www.tei-c.org/ns/1.0" ident="annotation" mode="add" module="linking" xmlns:sch="http://purl.oclc.org/dsdl/schematron">
   <desc versionDate="2020-07-31" xml:lang="en">represents an annotation following the <ref target="#WADM">Web
       Annotation Data Model</ref>.</desc>
+  <desc versiondate="2021-10-21" xml:lang="de" versionDate="2021-10-21">repräsentiert eine Annotation die dem <ref target="#WADM">Web Annotation Data Model</ref>.</desc>
```

Comment:
There are two `@versionDate` attributes: one with upper case 'D', the other with lower case 'd'.